src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -251,10 +249,7 @@ private static IntPtr GetSslMethod(bool isServer, long options)
                 }
             }
 
-            if (IntPtr.Zero == method)
-            {
-                throw CreateSslException("Failed to get SSL method");
-            }
+            Debug.Assert(IntPtr.Zero != method, "Failed to get SSL method");
```

Comment:
Seems like this could actually be zero if the static cctor failed to get any of these.  Shouldn't we still throw in that case?